### PR TITLE
Removes the armament token system that is now only being used by a whopping single ghost role in space

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
You know that armadyne guns vendor or whatever the hell it was called that spawns in sec every round that you have no use for? Yeah that thing, its the leftovers of an old system where sec could choose guns using special armament tokens or whatever, long story short we don't use it anymore and I'm not sure why we kept it outside of whatever replaced it just not bothering to.

## How This Contributes To The Skyrat Roleplay Experience
Kinda goofy that we have this machine spawn in sec but it holds zero actual purpose for anyone on station. We shouldn't be using it at all anymore.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
I don't remember where they spawned exactly on icebox but you can see for sure that it ain't here.

image
image
image

</details>

## Changelog

:cl:
del: The weapon token system, with associated vendor and tokens, has been removed because we don't actually use it anymore.
/:cl:

### CODE REVIEW

**Review by Zonespace27 — APPROVED** [review]

File: modular_skyrat/modules/sec_haul/code/misc/vending.dm [hunk 1]
```diff
@@ -20,7 +20,6 @@
 	contraband = list(
 		/obj/item/clothing/glasses/sunglasses = 2,
 		/obj/item/storage/fancy/donut_box = 2,
-		/obj/item/armament_token/sidearm_blackmarket = 2,
```
Comment (Zonespace27) [inline modular_skyrat/modules/sec_haul/code/misc/vending.dm:23]: actually, before merge, I *will* say that these had a use

Comment (Paxilmaniac) [reply modular_skyrat/modules/sec_haul/code/misc/vending.dm:23]: Damn shame

**Comment by xiangfei-chen** [comment]
I liked this. Gave us a cheap way to obtain lethal armaments in the event that we happened to need them. Shame to see it go.

**Comment by Zonespace27** [comment]
> I liked this. Gave us a cheap way to obtain lethal armaments in the event that we happened to need them. Shame to see it go.

You're free to add a similar system if you can give it a use

**Pull request merged**